.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# Description:
+
+# Tasks:
+
+- [ ] Describe the problem / feature
+- [ ] Write tests for it
+- [ ] Write code to solve the problem
```

Comment:
my intention is to show order, many developers jump in and fix a problem without writing a test to prove first there is a problem in first place - and I'm guilty of that one too, having explicit guide to write tests first might help.